plugins/depthCamera/src/DepthCameraDriver.cpp

```diff
@@ -120,23 +127,25 @@ bool GazeboYarpDepthCameraDriver::close()
     // Copy & paste like a Monkey
     if (this->m_updateImageFrame_Connection.get())
     {
-        m_imageCameraPtr->DisconnectNewImageFrame(this->m_updateImageFrame_Connection);
+        m_depthCameraPtr->DisconnectNewImageFrame(this->m_updateImageFrame_Connection);
         this->m_updateImageFrame_Connection = gazebo::event::ConnectionPtr();
     }
 
     if (this->m_updateRGBPointCloud_Connection.get())
     {
-        m_imageCameraPtr->DisconnectNewImageFrame(this->m_updateRGBPointCloud_Connection);
+        m_depthCameraPtr->DisconnectNewImageFrame(this->m_updateRGBPointCloud_Connection);
```

Comment:
I see, the method is already in the DepthSensor in Gazebo (see https://osrf-distributions.s3.amazonaws.com/gazebo/api/dev/classgazebo_1_1rendering_1_1DepthCamera.html#ab637c12f03aa1dc407d737a70b182ce5) but is not working? O_o